backend/endpoints/document_order.py

```
@@ -0,0 +1,80 @@
+    new_document_id = connect.get_body_arg("newDocumentId")
+    new_path = DocumentPath(new_document_id)
+    selected_document_id = connect.get_optional_body_arg("selectedDocumentId")
+
+    try:
+        document_name = get_document(api, new_path)["name"]
+    except:
+        raise ClientException("Failed to find a document to use.")
+
+    try:
+        latest_version = get_latest_version_path(api, new_path)
+    except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)